### ISSUE TITLE
Add support for a32x

### ISSUE THREAD

**Issue by azgaresncf** [openandroidinstaller-dev/openandroidinstaller#259]
**Device informations**
- Brand: Samsung
- Phone name: Galaxy A32 5G
- Device code (if unknown, leave blank): 
- Model (if several models exist): SM-A326B/DS

**Device images**  
Please provide links to any available images of the device, such as official stock firmware or custom ROMs you would like to install.  

Unfortunately custom ROMs are not yet available on this device.

**Do you own the device and would be willing to test the installer?**
- [X] (yes and yes)

**Additional context**  
Please provide any additional context or information that may be helpful in adding support for this device.

**Comment by MagicLike** [comment]
Probably blocked by #231 (?)

**Comment by azgaresncf** [comment]
> Probably blocked by #231 (?)

So is it why there isn't custom ROM or is it for the OAI process?

**Comment by MagicLike** [comment]
> So is it why there isn't custom ROM or is it for the OAI process?

No, this issue is related to OAI... Regarding not finding a ready to use ROM could be related, that not many people are flashing your exact device... Maybe you can look out for a build ROM on xda forums - sometimes there are some treasures to find there. Another option would be to request a build ROM from a project, if the allow it or try and build one yourself - though you will need some storage space for that.

**Comment by azgaresncf** [comment]
Thank you for this answer.

**Comment by MagicLike** [comment]
Why have you closed the issue?

**Comment by azgaresncf** [comment]
> Why have you closed the issue?

Because the Samsung Galaxy A32 doesn't have any custom ROM yet. So this is pretty useless I guess for rn

**Comment by MagicLike** [comment]
> > Why have you closed the issue?
> 
> Because the Samsung Galaxy A32 doesn't have any custom ROM yet. So this is pretty useless I guess for rn

That does not mean it is not useless. - As I said people can also build their own ROMs and stuff (Also OAI will have a `untested` tag in the next update)
